webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.module.scss

```diff
@@ -0,0 +1,10 @@
+.pivoter {
+  [role='tablist'] > div {
+    &::before {
+      box-shadow: none !important;
+    }
+    > div {
+      transform: translate(0) !important;
+    }
```

Comment:
**question:** is there a way to define the selectors to be more specific than what is being applied to avoid having to use `!important`?